src/saga-dense.h

```diff
@@ -122,72 +124,97 @@ Saga(Penalty&               penalty,
      unsigned&              n_iter,
      std::vector<unsigned>& return_codes,
      std::vector<double>&   losses,
-     const bool             debug) noexcept
+     const bool             debug,
+     const bool             cyclic,
+     const unsigned         B) noexcept
 {
   using namespace std;
-
+    
   double wscale = 1.0;
-
+    
```

Comment:
There's a lot of added white space / diff noise in this file. Can you configure your edit to automatically trim trailing white space?